OpenRobertaServer/staticResources/js/app/roberta/controller/progCode.controller.js

```diff
@@ -1,111 +1,104 @@
-define([ 'exports', 'message', 'log', 'util', 'guiState.controller', 'program.controller', 'program.model', 'blockly', 'codeflask', 'jquery' ], function(
-        exports, MSG, LOG, UTIL, GUISTATE_C, PROG_C, PROGRAM, Blockly, CodeFlask, $) {
-
-    const INITIAL_WIDTH = 0.5;
+define(["require", "exports", "message", "util", "guiState.controller", "program.controller", "program.model", "blockly", "codeflask", "jquery"], function (require, exports, MSG, UTIL, GUISTATE_C, PROG_C, PROGRAM, Blockly, CodeFlask, $) {
+    Object.defineProperty(exports, "__esModule", { value: true });
+    exports.setCodeLanguage = exports.setCode = exports.init = void 0;
+    var INITIAL_WIDTH = 0.5;
     var blocklyWorkspace;
     var flask;
     /**
-     * 
+     *
      */
     function init() {
         blocklyWorkspace = GUISTATE_C.getBlocklyWorkspace();
         flask = new CodeFlask('#codeContent', {
-            language : 'java',
-            lineNumbers : true,
-            readonly : true
+            language: 'java',
+            lineNumbers: true,
+            readonly: true
         });
         initEvents();
     }
     exports.init = init;
-
     function setCode(sourceCode) {
         flask.updateCode(sourceCode);
     }
     exports.setCode = setCode;
-
     function setCodeLanguage(language) {
         var langToSet;
         switch (language) {
-        case 'py':
-            langToSet = 'python';
-            break;
-        case 'java':
-            langToSet = 'java';
-            break;
-        case 'ino':
-        case 'nxc':
-        case 'cpp':
-            langToSet = 'clike';
-            break;
-        case 'json':
-            langToSet = 'js';
-            break;
-        default:
-            langToSet = 'js';
+            case 'py':
```

Comment:
is it true, that whitespaces are used here? I'd dislike tabs (as in python, where whitespaces are preferred, too)